src/iexamine.cpp

```diff
@@ -6330,6 +6343,8 @@ iexamine_functions iexamine_functions_from_string( const std::string &function_n
         iexamine_examine_function func = iter->second;
         if( function_name == "none" ) {
             return iexamine_functions{&iexamine::always_false, func};
+        } else if( function_name == "invalid" ) {
+            return iexamine_functions{&iexamine::false_and_debugmsg, func};
```

Comment:
Not part of this PR, but it would be nice to be a bit clearer about which iexamine functions actually return a value (without needing to look at the function itself or its specification in a header file).

Reply:
I'm actually thinking it'd be better to just make "is examinable" a method on iexamine_actor along with `call()` and rip out most of what I added in my previous PR - gonna try and take a stab at that this week.